golem/managers/proposal/plugins/negotiating/negotiating_plugin.py

```diff
@@ -12,17 +13,21 @@
 
 logger = logging.getLogger(__name__)
 
+DEFAULT_PROPOSAL_RESPONSE_TIMEOUT = timedelta(seconds=5)
```

Comment:
```suggestion
DEFAULT_PROPOSAL_RESPONSE_TIMEOUT = timedelta(seconds=5)
```

in our discussion with Darek today, he claimed that message exchanges between the nodes can take even more than a minute... :/

maybe we should consider raising such timeouts to e.g. 60 seconds in that case? (unsure, just reporting what Darek observed)

Reply:
I prefer having it at 5 for near future and they maybe increase it later